CONTRIBUTING.md

```diff
@@ -56,8 +55,8 @@ There is an npm build target available (`npm run dev-build`) linking all Langium
 A project you created with `yo langium` contains a dependency to `langium` (e.g. `0.1.0`) and a dev-dependency to `langium-cli`) inside `package.json` by default. Now, the easiest way to switch to your own build is to locally reference the required Langium packages folders.
 Let's assume your local Langium checkout resides on the same level as your own language project, then these two commands will set the references inside the `package.json`:
 ```shell
-npm install --save ../langium/packages/langium
-npm install --save-dev ../langium/packages/langium-cli
+npm install ../langium/packages/langium
+npm install -D ../langium/packages/langium-cli
```

Comment:
I figured out that can use `npm link -S langium` and `npm link -D langium-cli` to add those dependencies in the versions "installed"/"published" to the user global package cache. If a package is installed as link to a local folder via `npm link` the link target is then referenced in the `package.json`. Otherwise, the path of the package within the global package cache is used. If the particular package is not available in the global cache it will be downloaded from the public registry first.
In contrast to the statement in the `help` contents (seem to be equal for several commands) saying `--safe` is true by default, this is not the case for me. I had to state `--safe` or `-S` get the dependency added in the `package.json`.

I suggest to replace the above statements by the `npm link`-based ones as they appear to be more consistent with the preparations suggested above to me.

Reply:
What does -S and -D mean?

Reply:
-S: "add the package to the 'dependencies' list in the package.json" (since npm5 by default true for `npm install`)
-D: "add the package to the 'devDependencies' list in the package.json"

Reply:
I am pretty certain that you need the local reference for the VSCode debugger to work correctly in the VSCode workspace, but I may be in error here. We should check this before altering the description.

Reply:
```shell
npm link -S langium
npm link -D langium-cli
```
creates the local file references as desired (my local build was globally installed before (machine-wide). So, this can replace the other instruction then. @sailingKieler It seems you can only omit the options in case of `npm install` and not in case of `npm link`. I can confirm your observation.